### ISSUE TITLE
About the input of Model

### ISSUE THREAD

**Issue by lililuya** [anliyuan/Ultralight-Digital-Human#30]
Hi,  thanks for your great work!! I am concerned about the input of the Model, why we need to process the img  as following:
![image](https://github.com/user-attachments/assets/6f7c2176-a98d-42ca-a361-a22275cc7ec1)
(after resize to 168*168, why index from 4 to 164 ? And the explaination about the way to get mask.)
I would greatly appreciate it if you could clarify my doubts.

**Comment by anliyuan** [comment]
取4到164是为了后面inference的时候贴回去的时候不会违和。mask的时候留一点边界也是为了防止推理时候出现色差什么的

**Comment by lililuya** [comment]
> 取4到164是为了后面inference的时候贴回去的时候不会违和。mask的时候留一点边界也是为了防止推理时候出现色差什么的

Thank for your reply, I have tried the whole pipeline and  ecountered  a problem. I used a 3min video first train the syncnet, and chose the lowest loss model, then I train the unet model about 200 epoch. In inference stage, I found that some frames is not aligned with the audio, the mouth kept opened. Is the reason of the trained syncnet or some other ons?
+ the syncnet lowest loss is `5e-5`
![image](https://github.com/user-attachments/assets/e57e618e-078c-48bf-9a43-a118ff341ba8)
+ the unet is converged in  `8e-3`
![image](https://github.com/user-attachments/assets/385f91f3-236d-4eea-a8d0-bbe8fb88698f)
Looking for your reply! Thanks.

**Comment by anliyuan** [comment]
I found you have more then 7000 frame when train unet, but you said you are using a 3min video, maybe something wrong here?

**Comment by lililuya** [comment]
> I found you have more then 7000 frame when train unet, but you said you are using a 3min video, maybe something wrong here?

Yeah, sorry for that, that was a verbal mistake, I use about 4-5min video to train the unet,  so, is there a limitation of video clip?

**Comment by anliyuan** [comment]
You need to use same video when train syncnet and unet. Loss looks good, im not sure what goes wrong, you can sand your video to my email
